components/questions/QuestionOrSignIn.tsx

```diff
@@ -18,7 +18,7 @@ export function QuestionOrSignIn({
   const { data: session, status: authStatus } = useSession()
 
   const questionId = useQuestionId()
-  const qQuery = api.question.getQuestion.useQuery(
+  const qQuery = api.legacyQuestion.getQuestion.useQuery(
```

Comment:
As far as I can tell this is the only place that uses the API rather than makes a Prisma call directly. 

I've left it as the legacy router for convenience's sake, the `v1` API returns a slightly different type for `question`.